### PR TITLE
New version: Timthreetwelve.WUView version 0.5.14

### PR DESCRIPTION
**Pull request by ottnorml**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111589&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/t/Timthreetwelve/WUView/0.5.14/Timthreetwelve.WUView.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.8.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Timthreetwelve.WUView
+PackageVersion: 0.5.14
```
Comment (mdanish-kh) [inline manifests/t/Timthreetwelve/WUView/0.5.14/Timthreetwelve.WUView.installer.yaml:5]:
```suggestion
PackageVersion: 0.5.14.0
```

File: manifests/t/Timthreetwelve/WUView/0.5.14/Timthreetwelve.WUView.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Created with Komac v1.8.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Timthreetwelve.WUView
+PackageVersion: 0.5.14
```
Comment (mdanish-kh) [inline manifests/t/Timthreetwelve/WUView/0.5.14/Timthreetwelve.WUView.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.5.14.0
```

File: manifests/t/Timthreetwelve/WUView/0.5.14/Timthreetwelve.WUView.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with Komac v1.8.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Timthreetwelve.WUView
+PackageVersion: 0.5.14
```
Comment (mdanish-kh) [inline manifests/t/Timthreetwelve/WUView/0.5.14/Timthreetwelve.WUView.yaml:5]:
```suggestion
PackageVersion: 0.5.14.0
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
PackageVersion should match Control Panel (`appwiz.cpl`) / Apps&Features

After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/t/Timthreetwelve/WUView` should be renamed from `0.5.14` -> `0.5.14.0`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by ottnorml** [comment]
> PackageVersion should match Control Panel (`appwiz.cpl`) / Apps&Features
> 
> After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/t/Timthreetwelve/WUView` should be renamed from `0.5.14` -> `0.5.14.0`
> 
> A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.
> 
> After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

Thanks for the suggested changes and the instructions on how to implement them. I will have a look tomorrow.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**